retrievalmarket/types.go

```diff
@@ -120,6 +124,12 @@ type RetrievalPeer struct {
 	PieceCID *cid.Cid
 }
 
+// SignedRetrievalAsk is an ask signed by the miner's private key
+type SignedRetrievalAsk struct {
```

Comment:
@dirkmc  We don't really sign the QueryResponse for the query protocol. So, why are we signing the ask response here ?